fixing interactions between atoms with the same charge

lorenzo-22 opened the pull request:
fix #64 fixing interactions between atoms with the same charge by changing how "sets_sizes" list is created.

lorenzo-22 committed:
fixing interactions between atoms with the same charge

mtiberti reviewed (changes requested):
not much to review here :) cool that this was a small fix!

however, would you add a test to the dataset for same-charge interactions?

this means
  * generating reference data in `tests/data` of same-charge interactions
  * modifying `test_libinteract.py` so that a test on same-charge interactions is performed you can check e.g. `test_do_sb` and the objects/functions called there to see how this is done
 
besides, have you run the tests after you did your fix?

lorenzo-22 committed:
Add files via upload

added files for test

lorenzo-22 committed:
Update test_libinteract.py

modified test_libinteract.py so that a test on same-charge interactions is performed

mtiberti reviewed (changes requested):
can we change the file name of the data file to something that makes it clear that these are same-charge interactions? I also don't think these should be called salt_bridge since they are not salt bridges

lorenzo-22 committed:
Rename salt-bridges_all.csv to same_charge_all.csv
